src/functionaltests/java/com/ericsson/ei/scaling/TestScalingAndFailoverRunner.java

```diff
@@ -0,0 +1,14 @@
+package com.ericsson.ei.scaling;
+
+import org.junit.runner.RunWith;
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(features = "src/functionaltests/resources/features/scalingAndFailover.feature", glue = {
+        "com.ericsson.ei.scaling" }, plugin = {
+                "html:target/cucumber-reports/TestSubscriptionTriggerRunner" }, monochrome = false)
```

Comment:
monochrome = false, can be deleted in CucumberOptions annotation the monochrome set false as default so unneeded here